src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -44,15 +44,11 @@ public static ICapabilityStatementBuilder Create(IModelInfoProvider modelInfoPro
             EnsureArg.IsNotNull(modelInfoProvider, nameof(modelInfoProvider));
             EnsureArg.IsNotNull(searchParameterDefinitionManager, nameof(searchParameterDefinitionManager));
 
-            string manifestName = $"{typeof(CapabilityStatementBuilder).Namespace}.{modelInfoProvider.Version}.BaseCapabilities.json";
+            using Stream resourceStream = modelInfoProvider.OpenVersionedFileStream("BaseCapabilities.json");
+            using var reader = new StreamReader(resourceStream);
```

Comment:
nit: i think you should still keep the {} for the readability and also looks like the following line is misaligned?

Reply:
Ah, this one is ok, this is the C#8 `using` which assumes the parent scope